runners/spark/src/main/java/org/apache/beam/runners/spark/SparkRunner.java

```diff
@@ -142,6 +143,8 @@ private void registerMetrics(final SparkPipelineOptions opts, final JavaSparkCon
             : Optional.<CheckpointDir>absent();
     final Accumulator<NamedAggregators> aggregatorsAccumulator =
         SparkAggregators.getOrCreateNamedAggregators(jsc, maybeCheckpointDir);
+    // Instantiate metrics accumulator
+    MetricsAccumulator.getOrCreateInstance(jsc, maybeCheckpointDir);
```

Comment:
You're right, this method is now only called in this place, I'll rename and have it return void.